kaldb/src/main/java/com/slack/kaldb/logstore/search/KaldbDistributedQueryService.java

```diff
@@ -119,11 +119,13 @@ private void updateStubs() {
   }
 
   private List<SearchResult<LogMessage>> distributedSearch(KaldbSearch.SearchRequest request) {
+    LOG.info("Starting distributed search for request: {}", request);
```

Comment:
Is the string representation of the `request` readable, or are we better off pulling out the fields we're interested in directly?

Reply:
It's readable. Just logged a sample log line from unit tests. 

```
[INFO ] 2022-01-26 14:06:20.339 [main] KaldbLocalQueryService - Received search request: index_name: "testindex"
query_string: "Message1"
start_time_epoch_ms: 1601547000000
end_time_epoch_ms: 1601547010000
how_many: 10
bucket_count: 2
```

Reply:
We can start with this and see how it does.